### PR TITLE
Get started button tests

### PR DESCRIPTION
**Pull request by oponder**
Towards: https://github.com/giantswarm/happa/issues/759

We're already indirectly testing that the get started button shows up in the Getting Started tests. I added a test now to check that it goes away when the cluster is older than 30 days.

Also refactored to non-persisted nock calls.

### CODE REVIEW

File: __tests__/GettingStarted.js [hunk 1]
```diff
@@ -19,56 +18,36 @@ import {
...
+
+  await wait(() => expect(nock.isDone()).toBeTruthy());
+
+  nock.cleanAll();
 });
 
 /************ TESTS ************/
 
 it('lets me get there from the dashboard and go through the pages', async () => {
+  // prettier-ignore
```
Comment (axbarsan) [inline __tests__/GettingStarted.js]: Why is this needed?

File: __tests__/GettingStarted.js [hunk 2]
```diff
@@ -93,3 +72,22 @@ it('lets me get there from the dashboard and go through the pages', async () =>
...
+it('the get started button does not show up if the cluster is older than 30 days', async () => {
+  const futureDate = new Date();
+  futureDate.setMonth(futureDate.getMonth() + 2);
+
+  const clusterCreateDate2MonthsLater = futureDate.toISOString();
+
+  // prettier-ignore
+  getMockCall('/v4/clusters/', [Object.assign({}, v4ClustersResponse[0], {create_date: clusterCreateDate2MonthsLater})]);
+  // prettier-ignore
+  getMockCall(`/v4/clusters/${V4_CLUSTER.id}/`,Object.assign({}, v4AWSClusterResponse, {create_date: clusterCreateDate2MonthsLater,}));
```
Comment (axbarsan) [inline __tests__/GettingStarted.js]: Can you please extract these to separate variables?

File: __tests__/GettingStarted.js (hunk 1 above)
Comment (oponder) [reply __tests__/GettingStarted.js]: I like the kind of single scan line that forms from those getMockCall's being in a nice line... when prettier starts wrapping it all I get a bit sad.

But I can remove it and let prettier just do it's thing.
